catalog/spatial/csw/spatial-csw-source/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/source/TestCswSource.java

```diff
@@ -801,6 +897,27 @@ public void testRefresh() throws SecurityServiceException {
         Map<String, Object> configuration = new HashMap<>();
         cswSource.refresh(configuration);
 
+        cswSource = getCswSource(mockCsw, mockContext, "contentType");
+        CswSourceConfiguration defaultCswSourceConfiguration = getStandardCswSourceConfiguration(
+                "contentType",
+                "qname",
+                "queryprefix");
+
+        // Assert Defaults
+        assertDefaultCswSourceConfiguration(cswSource.cswSourceConfiguration,
```

Comment:
Is the purpose of this test to verify the outcome of a refresh with a specific configuration map? If so, I'm a bit confused by the previous two refresh calls since their outcomes aren't being verified. If there's more than one thing being tested here, they should be broken into their own methods.